cost/google/cud_report/Google_Committed_UseDiscount_Report.pt

```diff
@@ -0,0 +1,124 @@
+name "Google Committed Use Discount (CUD) Report"
+rs_pt_ver 20180301
+type "policy"
+short_description "Report on all Google CUDs that have been purchased. \n See the [README](https://github.com/rightscale/policy_templates/tree/master/cost/google/cud_report) and [docs.rightscale.com/policies](http://docs.rightscale.com/policies/) to learn more."
+long_description "Version: 1.0"
+category "Cost"
+severity "low"
+
+parameter "param_email" do
+  type "list"
+  label "Email addresses of the recipients you wish to notify"
+end
+
+parameter "param_google_project" do
+  type "string"
+  label "Google Cloud Project"
+  allowed_pattern /^[0-9a-z:\.-]+$/
+end
+
+parameter "param_google_status" do
+ label "CUD status"
+ type "string"
+ allowed_values "ALL","ACTIVE","EXPIRED"
+ default "ALL"
+end
+
+auth "auth_google", type: "oauth2" do
+  token_url "https://oauth2.googleapis.com/token"
+  grant type: "jwt_bearer" do
+  iss cred("GCE_PLUGIN_ACCOUNT")
+  aud "https://oauth2.googleapis.com/token"
+  additional_claims do {
+      "scope" => "https://www.googleapis.com/auth/compute"
+    } end
+  signing_key cred("GCE_PLUGIN_PRIVATE_KEY")
```

Comment:
Recommend remove PLUGIN from this cred name

Reply:
Renamed "GCE_PLUGIN_ACCOUNT and GCE_PLUGIN_PRIVATE_KEY" with "GC_SA_CLIENT_EMAIL and GC_SA_PRIVATE_KEY" respectively